common-utils/src/common/deployment/__init__.py

```diff
@@ -28,37 +24,25 @@
     PrivateAttr,
     StrictStr,
     conint,
-    validator,
 )
 from pydantic.error_wrappers import ValidationError
 from slugify import slugify
 
-# Create logger
+# Create logger for module
 LOGGER_NAME = __name__
 LOGGER = logging.getLogger(LOGGER_NAME)
 
 # script path for referencing bash scripts
 SCRIPT_PATH = os.path.dirname(os.path.realpath(__file__))
 
-# environment config for CI/CD
-ENVIRONMENT_TYPE = os.getenv("PREFECT_ENVIRONMENT_TYPE", "dev").lower()
-
 # config for deploying task definitions and flows
-DEPLOYMENT_TYPE = os.getenv("PREFECT_DEPLOYMENT_TYPE", "test").lower()
+DEPLOYMENT_TYPE = os.getenv("POCKET_PREFECT_DEPLOYMENT_TYPE", "dev").lower()
 GIT_SHA = os.getenv("CIRCLE_SHA1", "dev")[0:7]
 AWS_REGION = os.getenv("DEFAULT_AWS_REGION", "us-east-1").lower()
-
-# config for supporting using different pyproject.toml path when developing flows
-# running deployment cli with this set may result in errors
-# this is actually used in the flow deployment run_command script, because the prefect cli needs to run from the directory of the flow files
-CWD_DIR = os.path.join(os.getcwd(), "pyproject.toml")
-
-PYPROJECT_PATH = os.path.abspath(
-    os.path.expanduser(os.getenv("PREFECT_PYPROJECT_PATH", CWD_DIR))
-)
-
-# config to disable validation on FlowSpec for flow execution environments
-DISABLE_FLOW_SPEC = os.getenv("PREFECT_DISABLE_FLOW_SPEC", "false").lower()
```

Comment:
removing the pyproject.toml based validation from the FlowSpec pydantic validation.  This means that aws libraries are no longer required for anything other than using the deploy-cli for deploying docker images and flows.   This also removes the potential for FlowSpec model validation impacting flow run execution.

Getting rid of these globals is part of that change.